client/cmd/dexc-desktop/main.go

```diff
@@ -0,0 +1,303 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+/*
+dexc-desktop is the Desktop version of the DEX Client. There are a number of
+differences that make this version more suitable for less tech-savvy users.
+
+| CLI version                       | Desktop version                          |
+|-----------------------------------|------------------------------------------|
+| Installed by building from source | Installed with an installer program.     |
+| or downloading a binary.          | Debian archive for Debian Linux,         |
+|                                   | Inno Setup for Windows.                  |
```

Comment:
This row is curious to me.  The installer seems independent of the app format.  dcrinstall kinda sucks, but it exists, and there are Arch packages: https://aur.archlinux.org/packages/dcrd, https://aur.archlinux.org/packages/dcrdata, etc. (and I can make one for dcrdex).

Reply:
> This row is curious to me. The installer seems independent of the app format. dcrinstall kinda sucks, but it exists,

I see what you mean. There are different needs for a program that is expected to run without a terminal though. dcrinstall is an installer program, just with a different type of installation in mind. But the contrast I painted here is not quite right.

> and there are Arch packages: https://aur.archlinux.org/packages/dcrd, https://aur.archlinux.org/packages/dcrdata, etc. (and I can make one for dcrdex).

If we can get an Arch distro put together, that'd be awesome. I'm pretty sure that the "Desktop Entry" files and filepaths work exactly the same. Just gotta use some pacman thing to package it probably?

I can't say whether a mainstream Debian repository would accept the package, but even beginner Debian users know how to install a deb archive.

My next priority is Windows, I think.